_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Introduction to Mixing Audio - How to not sound like a Dial Up Modem"
+author: flavius
+categories: [ audio ]
+image: "mixer.webp"
+featured: false
+---
+What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio?
+How much does it cost?
+
+### - Introduction
+
+Audio mixing, what is it? Audio Mixing is where you take multiple input sources of audio and put them into a single or fewer number output signals. An example of input sources would be a microphone, game audio, stream alert audio. An example of an output signal would be our headphones, speakers, or stream. Can you achieve audio mixing for streams and videos without a digital or physical audio mixer? Mostly, but in practice it is significantly more complicated to route audio properly without using some form of audio mixing software or hardware when dealing with multiple sources.
+
+### - Goals of Audio Mixing
+
+The goals of audio mixing can be as simple as taking a microphone and game audio then pushing it to a single output. They can also be as elaborate as dividing out a microphone, or even two or three microphones, game audio, alerts, computer sounds, and music. Each having their own source to pull audio from then pushing that audio to two or more outputs, headphones, broadcast software such as OBS or SLOBS, or discord. Of course we don't want our stream audio going to friends on discord, and we don't want our mic audio going to our speakers if we aren't using headphones at the moment.
```

Comment:
How does someone "divide out" audio signals? Can you split up multiple microphones into multiple output signals from one combined input? If not, may want to rephrase the second sentence.

Reply:
The goals of audio mixing can be as simple as taking a microphone and game audio then pushing it to a single output. They can also be as elaborate as combining one or more microphones, game audio, alerts, computer sounds, and music. Each having their own source to pull audio from then pushing that audio to two or more outputs, headphones, broadcast software such as OBS or SLOBS, or discord. Of course we don't want our stream audio going to friends on discord, and we don't want our mic audio going to our speakers if we aren't using headphones at the moment.

Reply:
reworded to the above